tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -62,8 +57,6 @@ public void SetUp()
             _agentHealthReporter = Mock.Create<IAgentHealthReporter>();
 
             _defaultConfig = new TestableDefaultConfiguration(_environment, _localConfig, _serverConfig, _runTimeConfig, _securityPoliciesConfiguration, _bootstrapConfiguration, _processStatic, _httpRuntimeStatic, _configurationManagerStatic, _dnsStatic, _agentHealthReporter);
-
-            TestableDefaultConfiguration.ResetStatics();
```

Comment:
Did removing this cause the test coverage to drop?

Reply:
The `ResetStatistics()` method didn't do anything useful -- it set the value of a couple of private fields that weren't referenced anywhere else. I don't know what we intended for it to do, but it doesn't do that...

Reply:
It looks those fields can be removed. That TestableDefaultConfiguration was the only place left that they seemed to be used. But that can be done in a separate PR.